src/_zkapauthorizer/lease_maintenance.py

```diff
@@ -458,7 +457,7 @@ class LeaseMaintenanceConfig(object):
     min_lease_remaining: timedelta = attr.ib()
 
     @classmethod
-    def from_node_config(cls: Type[_T], node_config: Config) -> _T:
```

Comment:
Some SO post told me this would let you tell mypy that the method returns an instance of `cls` but mypy still complains so I just gave up on that idea.